build-tools/packages/build-cli/src/commands/generate/buildVersion.ts

```diff
@@ -34,7 +34,7 @@ export default class GenerateBuildVersionCommand extends BaseCommand<
 		}),
 		release: Flags.string({
 			description: "Indicates the build is a release build.",
-			options: ["release", "prerelease", "none"],
+			options: ["release", "prerelease", "prerelease - alpha", "prerelease - beta", "none"],
```

Comment:
I think the alpha/beta/public/untrimmed/default types setting should be separate from the release type. The "default" would be to publish whatever package.json says. The other options would set the types and update the version right before the pack stage in the pipeline.